test/test.py

```diff
@@ -1052,6 +1052,67 @@ def testCreateClientDefinitionJson(self):
             self.assertEqual(res.docs[0].json, '{"name":"henry"}')
             self.assertEqual(res.total, 1)
 
+    def testFieldsAsName(self):
+        conn = self.redis()
+
+        with conn as r:
+            r.flushdb()
+            if not check_version(r, 20200):
+                return
+
+            # create index
+            SCHEMA = (
+                TextField("$.name", sortable=True, as_name='name'),
```

Comment:
From the aesthetic perspective, isn't `TextField("$.name", sortable=True, as="name")` better?

Reply:
`as` is a saved word in python